.github/workflows/e2e-upgrade.yaml

```diff
@@ -128,6 +134,14 @@ jobs:
         run: |
           aws eks update-kubeconfig --name ${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}
           CLUSTER_NAME=${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }} INTERRUPTION_QUEUE=${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }} CLUSTER_ENDPOINT="$(aws eks describe-cluster --name ${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }} --query "cluster.endpoint" --output text)" TEST_SUITE="Integration" make e2etests
+
+          if [[ "$SOURCE" == 'aws' ]]; then
+            TEST_SUITE="Integration" ENABLE_METRICS=$ENABLE_METRICS \
+              CLUSTER_NAME="${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}" CLUSTER_ENDPOINT="$(aws eks describe-cluster --name ${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }} --query "cluster.endpoint" --output text)" \
+              INTERRUPTION_QUEUE="${{ steps.generate-cluster-name.outputs.CLUSTER_NAME }}" make e2etests
+          elif [[ "$SOURCE" == 'upstream' ]]; then
```

Comment:
What happens if we have something other than these two sources? Should we fail the test entirely? I'm sure stuff just isn't going to work if we keep proceeding

Reply:
The step would just pass, and proceed to clean up the cluster. I think failing the test would just give it a better signal